src/plugins/intel_gna/tests/functional/pass_tests/convolution_crop_axis_h.cpp

```diff
@@ -87,7 +87,9 @@ class CropAfterConvolutionTest : public testing::WithParamInterface<cccmParams>,
         const std::vector<int64_t> crop_end{20};
         const std::vector<int64_t> crop_stride{1};
         const std::vector<int64_t> axes{h_index_in_nchw};
+        OPENVINO_SUPPRESS_DEPRECATED_START
```

Comment:
I assume it would be better to refactor all builders including dead code in order to remove builders at all and not do work twice